test/runtests.jl

```diff
@@ -9,7 +9,7 @@ end
 
 @testset "Basics" begin
     x = malloc(Int, 10)
-    @test x isa AbstractVector{Int}
+    @test x isa DenseVector{Int}
```

Comment:
Still needs to be an AbstractVector.
```suggestion
    @test x isa AbstractVector{Int}
    @test x isa DenseVector{Int}
```